src/ert/scheduler/lsf_driver.py

```diff
@@ -261,7 +261,7 @@ async def submit(
         (Path(runpath) / LSF_INFO_JSON_FILENAME).write_text(
             json.dumps({"job_id": job_id}), encoding="utf-8"
         )
-        self._jobs[job_id] = (iens, QueuedJob(job_state="PEND"))
+        self._jobs[job_id] = (iens, QueuedJob(job_state="PEND"), int(time.time()))
```

Comment:
This tuple of 3 untyped elements might be a little to little self-documenting. An idea is to use pydantic objects but it might be too much again 🤷🏻

Reply:
I turned it into a pydantic class 👌
A tuple with two objects was fine, but I think three objects calls for a pydantic/data class.